python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -463,10 +465,23 @@ def _compute_psf(self, exposure, guess_psf=True):
         cell_set : `lsst.afw.math.SpatialCellSet`
             PSF candidates returned by the psf determiner.
         """
-        self.log.info("First pass detection with Guassian PSF FWHM=%s", self.config.install_simple_psf.fwhm)
+        def log_psf(msg):
```

Comment:
Just a preference, but I would like to see `exposure` (or `psf`) passed in explicitly, along with `background`.

Reply:
It's a closure, so it already has access to all local variables.